.buildkite/scripts/e2e/util.py

```diff
@@ -21,7 +21,13 @@ def call_url_with_retry(url: str, max_retries: int = 5, delay: int = 1) -> reque
 
 def get_logstash_container() -> Container:
     client = docker.from_env()
-    return client.containers.get("elastic-package-stack-e2e-logstash-1")
+    # `all=True` catches stopped ones as well, we need all to collect logs if container stops abnormally
+    containers = client.containers.list(all=True)
+    for container in containers:
+        print(f"Container name: {container.name}")
+        if "logstash-" in container.name:  # using only "logstash" may catch logstash ready one
```

Comment:
Is there a compromise between the specificity of `"elastic-package-stack-e2e-logstash-1"` and the potentially ambiguous `"logstash-"`? Maybe in the future there would be another `logstash-` container and this test could fail?

Reply:
> Is there a compromise between the specificity of "elastic-package-stack-e2e-logstash-1" and the potentially ambiguous "logstash-"?

By default LS container gets `"elastic-package-stack-e2e-logstash-1"` naming but we didn't cover some cases: I have seen BK agent path attached to the container name and/or LS container didn't run due to internal error. With `client.containers.list(all=True)` stopped containers also will be listed and when accessed error clearly tells cannot access to stopped container. If naming is different we still will be able to resolve with `logstash-` [prefix attached by `elastic-agent`](https://github.com/elastic/elastic-package/blob/main/internal/install/application_configuration.go#L109). 

> Maybe in the future there would be another logstash- container and this test could fail?

So far, a single LS container will be spinned by design and if we will have a multiple instances, we need to clarify the use case first and what would be goal I think.